packages/concerto-core/lib/decoratormanager.js

```diff
@@ -253,6 +253,178 @@ class DecoratorManager {
         newModelManager.fromAst(decoratedAst);
         return newModelManager;
     }
+
+    /**
+     * Applies all the decorator commands from the DecoratorCommandSet
+     * to the ModelManager.
+     * @param {*} decoratorCommandSet the DecoratorCommandSet object
+     * @returns {Object} a new model manager with the decorations applied
+     */
+    static getDecoratorMaps(decoratorCommandSet) {
```

Comment:
I think that this should be a private function?

Reply:
Done